src/gui/render/data_structures/frame_buffer_multisample.hpp

```diff
@@ -0,0 +1,160 @@
+// -*- lsst-c++ -*-
+/*
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ */
+
+/**
+ * @file frame_buffer_multisample.hpp
+ *
+ * @author @AlemSnyder
+ *
+ * @brief Defines FrameBufferMultisample
+ *
+ * @ingroup GUI  DATA_STRUCTURES
+ *
+ */
+
+#pragma once
+
+#include "../../../types.hpp"
+
+#include <GL/glew.h>
+#include <GLFW/glfw3.h>
+
+namespace gui {
+
+namespace data_structures {
+
+/**
+ * @brief Handles Frame Buffer
+ *
+ * @details Creates and handles a GPU frame buffer. Creates color texture,
+ * and depth texture.
+ *
+ */
+class FrameBufferMultisample {
+ private:
+    GLuint frame_buffer;
+    GLuint render_texture;
+    GLuint depth_buffer;
+    GLuint frame_buffer_single;
+    GLuint render_texture_single;
+    screen_size_t width_;
+    screen_size_t height_;
+    uint32_t samples_;
+
+ public:
+    /**
+     * @brief Delete constructor method
+     *
+     * @param FrameBufferMultisample
+     */
+    FrameBufferMultisample(const FrameBufferMultisample& obj) = delete;
```

Comment:
You also now need to delete/define

- [ ] Move CTOR
- [ ] Copy assignment
- [ ] Move assignment